jekyll/_cci2/install-machine-runner-3-on-linux.adoc

```diff
@@ -3,17 +3,14 @@ contentTags:
   platform:
   - Cloud
 ---
-= Install machine runner 3.0 on Linux - Open preview
```

Comment:
I addressed this in relation to your comment on https://github.com/circleci/circleci-docs/pull/8438#discussion_r1386544890 and have some other doc fixes in there (like de-dupping package install instructions), it may be worth doing a checkout on this file to avoid merge conflicts.

Reply:
I'll closed this PR.